README.md

```diff
@@ -43,21 +27,63 @@ miscellaneous tasks helpful to support personnel.
 * `setup-maas.yml` - deploys, sets up, and installs Rackspace
 [MaaS](http://www.rackspace.com/cloud/monitoring) checks
 for Rackspace Private Clouds.
-* `setup-logging.yml` - deploys and configures Beaver, Logstash,
+* `setup-logging.yml` - deploys and configures Filebeat, Logstash,
 Elasticsearch, and Kibana to tag, index, and expose aggregated logs from all
 hosts and containers in the deployment using the related plays mentioned
 above.
-* `site.yml` - deploys all the playbooks mentioned here.
 * `verify-maas.yml` - confirms each maas check selected for each host has been
-captured server-side for recording in MaaS and that each check has at least one
-alarm configured for it.
+captured server-side, for recording in MaaS and that each check has at least one
+alarm configured for it. This playbook can also be used to perform local checks
+and ignore remote api checks by specifing ``maas_use_api: False`` in the
+``user_rpco_variables_overrides.yml`` file.
+* `ceph-all.yml` - Runs the `ceph-mon.yml` and `ceph-osd.yml` playbooks
+* `ceph-mon.yml` - Runs the `ceph.ceph-mon` Ansible role, which is an external role
+  located at https://github.com/ceph/ansible-ceph-mon to deploy the ceph monitor bits
+* `ceph-osd.yml` - Runs the `ceph.ceph-osd` Ansible role, which is an external role
+  located at https://github.com/ceph/ansible-ceph-osd to deploy the ceph osd bits
+
 
-# Basic Setup:
+# Quick Start with an RPCO All-In-One(AIO)
+
+To build an AIO, first clone the RPCO repo:
+```
+cd /opt && git clone --recursive https://github.com/rcbops/rpc-openstack
+```
+
+Set the ``DEPLOY_AIO`` variable:
+```
+export DEPLOY_AIO='yes'
+```
+
+Run the ``deploy.sh`` script. It is recommended to run this script in either
+a tmux or screen session. It will take about 90min to complete:
+```
+cd /opt/rpc-openstack
+./scripts/deploy.sh
+```
+
+## Deploying MaaS with an AIO
+1. Add the following variables to`/etc/openstack_deploy/user_rpco_variables_overrides.yml`:
+```
+rackspace_cloud_tenant_id: YourTenantID
+rackspace_cloud_username: YourUsername
+rackspace_cloud_api_key: YourAPIKey
+```
+2. Run the MaaS setup plays:
+ `cd /opt/rpc-openstack/rpcd/playbooks && openstack-ansible setup-maas.yml`
+3. Run the MaaS verify play:
+ `cd /opt/rpc-openstack/rpcd/playbooks && openstack-ansible verify-maas.yml`
+     MaaS Verification _may_ fail if executed within the first few moments after
+     the Setup plays complete as the MaaS service registers checks and alarms
```

Comment:
this sentence is a little long and confusing. could you please restate? Does it fail because it is still registering checks and alarms?